src/components/TextField/TextField.tsx

```diff
@@ -49,7 +51,7 @@ export const TextField: React.FC<Props> = ({
       </div>
 
       {hasError && (
-        <p className="help is-danger">{`${label} is required`}</p>
+        <p className="help is-danger">{textError ? label + textError : `${label} is required`}</p>
```

Comment:
```suggestion
        <p className="help is-danger">{textError && `${label + textError}` </p>
```
pay attention that you always have some value in `textError`, so this check can be redutant